README.md

```
@@ -210,6 +212,8 @@ HTML Minifier Next: Ignoring `conservativeCollapse`—use with `collapseWhitespa
 | `preserveLineBreaks` | `collapseWhitespace` |
 | `removeEmptyElementsExcept` | `removeEmptyElements` |
 | `removeUnusedCSS` | `minifyCSS`, and not [a function of your own](#unused-css-removal) |
+| `shouldMinifyCSS` | `minifyCSS`, and not [a function of your own](#css-minification) |
+| `shouldMinifyJS` | `minifyJS`, and not [a function of your own](#js-minification) |
```

markdownlint-cli2 (0.23.2)

[warning] Link fragments should be valid

(MD051, link-fragments)